rbc/omnisci_backend/omnisci_buffer.py

```diff
@@ -195,6 +225,9 @@ def codegen(context, builder, signature, args):
 def omnisci_buffer_setitem_(typingctx, data, index, value):
     sig = types.none(data, index, value)
 
+    eltype = data.members[0].dtype
+    nb_value = value
```

Comment:
I'd like to use `data.eltype` here but `BufferType` do not hold this information